addons/mail/wizard/mail_wizard_invite.py

```diff
@@ -7,77 +7,43 @@
     _name = 'mail.wizard.invite'
     _description = 'Invite wizard'
 
-    @api.model
-    def default_get(self, fields):
-        result = super(Invite, self).default_get(fields)
-        if 'message' not in fields:
-            return result
-
-        user_name = self.env.user.display_name
-        model = result.get('res_model')
-        res_id = result.get('res_id')
-        if model and res_id:
-            document = self.env['ir.model']._get(model).display_name
-            title = self.env[model].browse(res_id).display_name
-            text = _(
-                '%(user_name)s invited you to follow %(document)s document: %(title)s',
-                user_name=user_name,
-                document=document,
-                title=title,
-            )
-        else:
-            text = _('%(user_name)s invited you to follow a new document.', user_name=user_name)
-
-        result['message'] = Markup("<div><p>%s</p><p>%s</p></div>") % (
-            _('Hello,'),
-            text,
-        )
-        return result
-
     res_model = fields.Char('Related Document Model', required=True, help='Model of the followed resource')
     res_id = fields.Integer('Related Document ID', help='Id of the followed resource')
     partner_ids = fields.Many2many('res.partner', string='Recipients')
     message = fields.Html('Message')
-    notify = fields.Boolean('Send Notification', default=True)
+    notify = fields.Boolean('Notify Recipients', default=True)
 
     def add_followers(self):
         if not self.env.user.email:
             raise UserError(_("Unable to post message, please configure the sender's email address."))
-        email_from = self.env.user.email_formatted
         for wizard in self:
-            Model = self.env[wizard.res_model]
-            document = Model.browse(wizard.res_id)
-
-            # filter partner_ids to get the new followers, to avoid sending email to already following partners
-            new_partners = wizard.partner_ids - document.sudo().message_partner_ids
-            document.message_subscribe(partner_ids=new_partners.ids)
-
-            model_name = self.env['ir.model']._get(wizard.res_model).display_name
-            # send a notification if option checked and if a message exists (do not send void notifications)
-            if wizard.notify and wizard.message and not is_html_empty(wizard.message):
-                message_values = wizard._prepare_message_values(document, model_name, email_from)
-                message_values['partner_ids'] = new_partners.ids
+            document = self.env[wizard.res_model].browse(wizard.res_id)
+            document.message_subscribe(partner_ids=wizard.partner_ids.ids)
+            if wizard.notify:
+                model_name = self.env['ir.model']._get(wizard.res_model).display_name
+                message_values = wizard._prepare_message_values(document, model_name, wizard.partner_ids.ids)
```

Comment:
as method is called on wizard, not sure why you give partners parameter ? Can be found using self.partner_ids in the method

Reply:
Oops, totally missed that one. We kind of stepped down from most of the changes.